scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/MockBigQuery.scala

```diff
@@ -44,7 +46,11 @@ object MockBigQuery {
  * Use [[mockTable(original:String)* mockTable]] to feed data into live BigQuery service and
  * [[queryResult]] to query them.
  */
-class MockBigQuery private (private val bq: BigQuery) {
+class MockBigQuery private (
+  private val bq: BigQuery,
+  stagingProject: Option[String] = None,
```

Comment:
I don't think we need these `None` defaults because they're already covered by the constructor you added in the `MockBigQuery` companion object.